tests/e2e/batch_presigned_exit_message.rs

```diff
@@ -0,0 +1,155 @@
+use assert_cmd::prelude::*;
+use std::process::Command;
+use types::SignedVoluntaryExit;
+
+/**
+
+Command sequence to verify signature:
+
+./target/debug/eth-staking-smith existing-mnemonic \
+       --chain mainnet \
+       --num_validators 3 \
+       --mnemonic 'ski interest capable knee usual ugly duty exercise tattoo subway delay upper bid forget say'
+./target/debug/eth-staking-smith existing-mnemonic \
+       --chain mainnet \
+       --num_validators 3 \
+       --mnemonic 'ski interest capable knee usual ugly duty exercise tattoo subway delay upper bid forget say'
```

Comment:
Is this a duplicate?

Reply:
Fixed in https://github.com/ChorusOne/eth-staking-smith/pull/59/commits/9f4121fe75ac9f7a1da83086fb20c66d9af2b67f